src/components/AddBookForm.js

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components/macro'
+
+import Button from './RectangleButton'
+import downArrow from '../assets/arrow-down.svg'
+
+export default function AddBookForm({ onCreateBook, onClickDownButton }) {
```

Comment:
I'd suggest to rename the `onCreateBook` function. We use the prefix `on*` for an eventlistener or s.th. that triggers another function. In this case, it is not the actual function per se, it is only used to pass the arguments to your `addBook` function, but it does not listen for an event. To make things easier, I'd just call it `addBook` as well and then in your App. js:
```
<AddBookForm
          addBook={addBook} 
...
```